src/pow.cpp

```diff
@@ -355,12 +355,16 @@ void multinomial_coefficients_mpz(int m, int n, map_vec_mpz &r)
 RCP<const Basic> pow_expand(const RCP<const Pow> &self)
 {
     RCP<const Basic> _base = expand(self->base_);
+    bool negative_pow = false;
     if (! is_a<Integer>(*self->exp_) || ! is_a<Add>(*_base))
         return self;
```

Comment:
I get a similar error as I was getting earlier:
https://gist.github.com/shivamvats/87f24b310da644a43bab

Is it because of my commits? I'm unable to test the counter example.